explainer.md

```diff
@@ -244,6 +244,8 @@ let gl = glCanvas.getContext("webgl", { xrCompatible: true });
 
 Ensuring context compatibility with an XR device through either method may have side effects on other graphics resources in the page, such as causing the entire user agent to switch from rendering using an integrated GPU to a discrete GPU.
 
+> **Note:** The `XRWebGLLayer` uses a WebGL context created by a Canvas element or `OffscreenCanvas` rather than creating it's own to both allow for the same content to be rendered to the XR device and the page, as well as allowing the page to load it's WebGL resorces prior to the session being created.
```

Comment:
```suggestion
> **Note:** The `XRWebGLLayer` uses a WebGL context created by a Canvas element or `OffscreenCanvas` rather than creating its own to both allow for the same content to be rendered to the XR device and the page, as well as allowing the page to load it's WebGL resources prior to the session being created.
```